### PR TITLE
Added accessibility documentation in style guide

### PR DESCRIPTION
**Pull request by stephherbers**
## Summary
Adding to the docs of some new resources we gathered with the web accessibility improvement effort. Also linked the doc Charlie created that has a ton of resources and CommCare-specific accessibility changes.

## Safety Assurance

- [X] Risk label is set correctly
- [X] All migrations are backwards compatible and won't block deploy
- [X] The set of people pinged as reviewers is appropriate for the level of risk of the change
- [X] If QA is part of the safety story, the "Awaiting QA" label is used
- [X] I have confidence that this PR will not introduce a regression for the reasons below

### QA Plan
no qa

### Safety story
minor-- just affects the styleguide

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### CODE REVIEW

File: corehq/apps/styleguide/templates/styleguide/_includes/atoms/accessibility.html [hunk 1]
```diff
@@ -98,6 +98,34 @@ <h1 id="atoms-accessibility">Accessibility</h1>
     <li><a href="https://a11yproject.com/">https://a11yproject.com/</a></li>
     <li><a href="http://www.carbondesignsystem.com/guidelines/accessibility">http://www.carbondesignsystem.com/guidelines/accessibility</a></li>
     <li><a href="https://www.w3.org/WAI/tutorials">https://www.w3.org/WAI/tutorials</a></li>
+    <li><a href="https://www.youtube.com/playlist?list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g">A11ycast videos</a></li>
+  </ul>
+  </p>
+    <p>
+    In 2020, USH began a web accessibility improvement effort, specifically surrounding screen readers.
+    <a href="https://developers.google.com/web/tools/chrome-devtools/accessibility/reference">This document</a> details
```
Comment (orangejenny) [inline corehq/apps/styleguide/templates/styleguide/_includes/atoms/accessibility.html:106]: This url looks like it might not be right.

**Comment by orangejenny** [comment]
Do you know how the style guide examples measure up, accessibility-wise? I'm especially thinking of the widgets like pagination, and the content on forms, and I'm guessing they're less accessible than web apps now is. Those might be high-value places to update, so we can point people at them in future code reviews.

**Review by orangejenny — APPROVED** [review]
Fine to merge assuming the one URL is either fine or gets fixed.

File: corehq/apps/styleguide/templates/styleguide/_includes/atoms/accessibility.html (hunk 1 above)
Comment (stephherbers) [reply corehq/apps/styleguide/templates/styleguide/_includes/atoms/accessibility.html]: good catch!

**Comment by stephherbers** [comment]
> Do you know how the style guide examples measure up, accessibility-wise? I'm especially thinking of the widgets like pagination, and the content on forms, and I'm guessing they're less accessible than web apps now is. Those might be high-value places to update, so we can point people at them in future code reviews.

This is a great point. Just by using the keyboard, I think a lot of it holds up, with a big exception that it skipped over the table in the pagination of the results. I'll look into this further and do a separate PR if necessary.

**Review by orangejenny — APPROVED** [review]

**Pull request merged**